c/recovery.c

```diff
@@ -17,12 +17,14 @@
 #include <metal/stdio.h>
 #include <metal/stdlib.h>
 #include <metal/string.h>
```

Comment:
@JoeNemo can you include `#include <metal/stdint.h>`? Otherwise Metal won't compile because `inttypes.h` uses `#include <stdint.h>` which results in including the LE version because of how our build scripts are written...